src/pandas_profiling/model/correlations.py

```diff
@@ -89,10 +89,6 @@ def calculate_correlation(
     Returns:
         The correlation matrices for the given correlation measures. Return None if correlation is empty.
     """
-
-    if len(df) == 0:
```

Comment:
is it in purpose or a wrong merge?

Reply:
this breaks spark integration, so it was removed from here and place changed. 

Spark Dataframe has no len method